install/upgrade-clickhouse.sh

```diff
@@ -0,0 +1,28 @@
+echo "${_group}Upgrading Clickhouse ..."
+
+# First check to see if user is upgrading by checking for existing clickhouse volume
+if [[ -n "$(docker volume ls -q --filter name=sentry-clickhouse)" ]]; then
```

Comment:
Wonder if it's worth adding a test for this? Should (hopefully) be pretty straightforward with the new python-based testing regime?

Reply:
Editing the install script is not easily tested using the new python tests. Those are effective after self-hosted sentry is installed and up and running. The new self-hosted Sentry upgrade tests I added sufficiently covers this IMO, it skips over this portion of the code on initial install and hits this on the upgrade